people_tracking_filter/CMakeLists.txt

```
@@ -51,14 +51,15 @@ add_library(people_tracking_filter
             src/tracker_kalman.cpp
             src/detector_particle.cpp
 )
+add_dependencies(people_tracking_filter people_msgs_generate_messages_cpp)
```

I believe this is the [suggested formatting](https://docs.ros.org/api/catkin/html/howto/format1/cpp_msg_dependencies.html)
```suggestion
add_dependencies(people_tracking_filter ${catkin_EXPORTED_TARGETS})
```